packages/core/admin/server/validation/project-settings.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { yup, validateYupSchemaSync } = require('@strapi/utils');
+
+const MAX_IMAGE_WIDTH = 750;
+const MAX_IMAGE_HEIGHT = MAX_IMAGE_WIDTH;
+const MAX_IMAGE_FILE_SIZE = 1024 * 1024; // 1Mo
+const ALLOWED_IMAGE_FILE_TYPES = ['image/jpeg', 'image/png', 'image/svg+xml'];
```

Comment:
Nope, just neved did it before so I didn't think about it, but it would be great, if we do so we should use this config on the front-end too @ronronscelestes wdyt?
Thanks for highlighting this JS

Reply:
It is not planed to have it configurable for now on the product side, and if we decide to do so it will be on a next iteration of the feat

Reply:
Alright, thanks for the input